modules/ding_variables/ding_variables.module

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * @file
+ * Define modules hooks.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function ding_variables_menu() {
+  $items = array();
+
+  $items['ding/variables'] = array(
+    'title' => 'Display variables as JSON output',
+    'access callback' => TRUE,
+    'delivery callback' => 'drupal_json_output',
+    'page callback' => 'ding_variables_display',
+  );
+
+  return $items;
+}
+
+/**
+ * Provide variables to the json output.
+ *
+ * @return array
+ *   List of all variables to the output.
+ */
+function ding_variables_display() {
+  // TODO: Find a way to remove hardcode.
+  $carousels = array(
+    'carousel',
+  );
+  $carousel_configs = array();
+  foreach ($carousels as $carousel) {
+    $carousel_configs[] = _ding_variables_get_carousel_configs($carousel);
+  }
+
+  $variables = array(
+    'ting_search_profile',
+    'ting_search_results_per_page',
+    'ting_search_result_message_limit',
+    'ting_search_url',
+    'ting_sort_default',
+    'ting_ranking_custom',
+    'ting_ranking_fields',
+    'ting_boost_fields',
+    'ting_agency',
+  );
+  $variables_configs = array();
+  foreach ($variables as $variable) {
+    $variables_configs[$variable] = variable_get($variable, '');
+  }
+
+  return array(
+    'variables' => $variables_configs,
+    'carousels' => $carousel_configs,
+  );
+}
+
+/**
+ * Get panels configurations.
+ *
+ * @return array
+ *   List of carousels pane configurations.
+ */
+function _ding_variables_get_carousel_configs($carousel) {
+  $query = db_select('panels_pane', 'pp');
+  $result = $query
+    ->fields('pp', array('subtype', 'configuration'))
+    ->condition('subtype', $carousel)
+    ->execute()
+    ->fetchAll();
```

Comment:
This seems to assume that #1007 is merged. Is that true?

Reply:
As I know we are still waiting for this to be merged.

Reply:
An that PR is firste set to next release and therefor we can't use this before that. I have contact DDB in regards to this issue.

Reply:
Ok, let us know if we need to refactor to current release.

Reply:
Will do, awaiting Gitte and Rolf

Reply:
@guddo after talk with Gitte: We would like the carousels export to use the current data structure as the changes to the workflow (moving them into panes) is not set for this release. But the app has to be released before that release. 

Short version: Yes, please refactor.